tutorials/milvus_lite_arize.ipynb

```diff
@@ -0,0 +1,1428 @@
+{
```

Comment:
can we name this file a bit more "use-case" centric? Arize keyword is not needed as it's confusing (this is phoenix) but also because it's redundant